test/relations.integration.js

```diff
@@ -882,11 +882,15 @@ describe('relations - integration', function() {
         { properties: { name: 'string' },
           dataSource: 'db',
           plural: 'todo-lists',
+          forceId: false,
         }
       );
       var todoItem = app.model(
         'todoItem',
-        { properties: { content: 'string' }, dataSource: 'db' }
+        { properties: { content: 'string' },
+          dataSource: 'db',
+          forceId: false,
```

Comment:
and here too.

BTW the indentation is off here too, both L891 an L892 are indented too much.